### ISSUE TITLE
Paper outline

### ISSUE THREAD

**Issue by b-biswas** [b-biswas/MADNESS#46]
[paper](https://www.overleaf.com/project/64009ae5ca701c410502bbf9)
Opening this issue to exchange on what we want to study/plot in the paper for MADNESS.
this is largely based on the issue opened for Debvader, some modifications are still to be made.

This paper aims to use the ground truth position information of the parametric/"real" simulation to evaluate the performance of the MADNESS deblender.

Primary references to move forward (please feel free to suggest more):
[Bastien's VAE paper](https://arxiv.org/abs/2005.12039) 
[Francois's Galaxy simulation paper](https://arxiv.org/abs/2008.03833)

Paper plan (preliminary):

- [ ] Introduction
  - [ ] Challenge of deblending
  - [ ]  Other deblenders to discuss... 
    - [ ] SDSS
    - [ ] SCARLET
    - [ ] GAN based deblender
    - [ ] Bastien's VAE
    - [ ] Recurrent Neural networks
  - [ ] intro to MADNESS 

- [ ] Dataset
  - [ ] Define the COSMOS dataset, explain the cut
  - [ ] Parametric models / "real galaxies"
  - [ ] Talk about BTK, PSF, noise 
  - [ ] simulations of isolated vs blended galaxy [`Fig: show examples of simulations`]

- [ ] Method
  - [ ] Variational Inference + VAEs
  - [ ] Normalizing flows
  - [ ] Deblender Implementation/Choice of architecture [`Fig: Architecture of models`] + training [`Fig: Simulations with VAE`] [`Fig: compare NF space with actual latent space`]
  - [ ] Optimization loss function

`Maybe we should merge the metrics and results section?`
- [ ] Metric (here we only describe how the metric is built)
  - [ ] Check literatture (Scarlet, BLISS?, MOF in DES/metadetect)
  - [ ] Pixel error? [`SCARLET vs MADNESS, pixel correlation plot`]
  - [ ] Bias in flux/color -> impact on redshift (compare measured flux by SExtractor with and without deblending) [`distribution of relative flux error -> fit with gaussian??`]
  [`Compare photometry results against scarlet and non-blended scenes`]
  - [ ] Bias in size -> impact on morphology (same) [`show plots for size, SNR and effects on flux reconstruction]
  - [ ] Bias in shear recovery (see the project with Thomas) [`Maybe compute moments? shear is probably out of scape here`]
  - [ ] Comparison with Scarlet / SExtractor / (other ?) (I am adding SExtractor because it more or less comes for free..)
      Previously mentioned metrics
  - [ ] benchmarking for Speed [`Plot time taken against the number of galaxies.`]
      How do we reliably measure the speed? (this is very important if we want people to believe the results)

- [ ] Results (here we show the results of the metric) + comparison
  - [ ] Simulation of Isolated galaxies
  - [ ] Pixel error
  - [ ] Flux error
  - [ ] size error
  - [ ] SNR dependence

- [ ] Conclusion and Discussion: 
  - [ ] A hierarchical PSF model
  - [ ] Masking and dealing with real data
  - [ ] choice of detection algorithms in the future scope of work
  - [ ] Obtaining the full posterior

Improvement on science
What is the best solution for deblending in terms of speed AND accuracy?
